dags/atd_metrobike_trips.py

```diff
@@ -0,0 +1,78 @@
+# Test locally with: docker compose run --rm airflow-cli dags test atd_metrobike_trips
+
+from datetime import datetime, timedelta
+
+from airflow.decorators import task
+from airflow.models import DAG
+from airflow.operators.docker_operator import DockerOperator
+
+from utils.slack_operator import task_fail_slack_alert
+
+default_args = {
+    "owner": "airflow",
+    "description": "Fetch metrobike trip data from dropbox and publish to Socrata",
+    "depend_on_past": False,
+    "start_date": datetime(2020, 12, 31),
+    "email_on_failure": False,
+    "email_on_retry": False,
+    "retries": 1,
+    "retry_delay": timedelta(minutes=5),
+    "on_failure_callback": task_fail_slack_alert,
+}
+
+docker_image = "atddocker/atd-metrobike:production"
```

Comment:
For ARM users (macs), this image does not pull and run because it doesn't have a manifest for ARM64. Are you building this by hand or with CI? Can you build this with ARM64 support, or adjust the CI to do it? LMK if I can help - i have some recipes for this I can share with you. I can't test this locally (on my mac, at least) yet.

Reply:
This was built by the CI in [atd-metrobike](https://github.com/cityofaustin/atd-metrobike). So that's the dockerfile that needs some TLC then we should be able to get you to run this locally. Frustrating but I guess that's the tech debt we have on some of these older docker images.

Reply:
Totally! Thanks for the info on where this is built. I'm going to make an issue or maybe a micro PR to Mac this image ma friendly. Thanks!!

Reply:
I'm on it! https://github.com/cityofaustin/atd-metrobike/pull/6